src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -733,10 +734,22 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		};
 	}
 
-	private rebuildActionBar() {
+	public rebuildActionBar() {
```

Comment:
do you really need to make this public? you could just call show/hide actionbar methods you added below, right?

Reply:
Moved the action bar changes its own PR: https://github.com/microsoft/azuredatastudio/pull/16178